cli/src/semgrep/formatter/sarif.py

```diff
@@ -275,16 +279,33 @@ def _rule_to_sarif(rule: Rule) -> Mapping[str, Any]:
             rule_json = {
                 "id": rule.id,
                 "name": rule.id,
-                "shortDescription": {"text": rule.message},
+                "shortDescription": {"text": f"Semgrep Finding - {rule.id}"},
                 "fullDescription": {"text": rule.message},
                 "defaultConfiguration": {"level": severity},
                 "properties": {"precision": "very-high", "tags": tags},
             }
 
         rule_url = rule.metadata.get("source")
+        references = []
+
         if rule_url is not None:
             rule_json["helpUri"] = rule_url
-
+            references.extend([f"[Semgrep Rule]({rule_url})"])
+
+        if "references" in rule.metadata:
```

Comment:
I'd go even further and check for truthiness (e.g. `if rule.metadata.get("references")`)